guides/source/active_record_postgresql.md

```diff
@@ -15,6 +15,8 @@ After reading this guide, you will know:
 * How to implement exclusion constraints.
 * How to implement full text search with PostgreSQL.
 * How to back your Active Record models with database views.
+* How to use Explain options to gain insights that could potentially further
+  improve PostgreSQL's query performance
```

Comment:
I'm not sure this warrants a bullet point here, this isn't meant to be a table of contents but more of an overview of the high level concepts in the guide. It's already probably a bit too specific.

Reply:
Happy to remove that @zzak & just did :) I was originally interpreting that each point in this section represents a part of the table of contents of the guide as a whole and I was attempting to maintain that uniformity(thus I originally added it). Also agree that it sounds more specific  👍 

P.S: My apologies for committing `wrap_text.md` by mistake and thank you so much for removing it :)